app/controllers/support_interface/ucas_matches_controller.rb

```diff
@@ -14,5 +14,21 @@ def process_match
       flash[:success] = 'Match marked as processed'
       redirect_to support_interface_ucas_match_path(match)
     end
+
+    def record_initial_emails_sent
+      match = UCASMatch.find(params[:id])
+
+      if match.update!(
+        matching_state: 'initial_emails_sent',
```

Comment:
Will this be overridden if new matching data arrives for this candidate after the emails were sent?